src/components/NormalLeftRow/index.jsx

```diff
@@ -262,10 +269,17 @@ NormalLeftRow.propTypes = {
    * Only necessary for ref rows.
    */
   setSchemaTree: func,
+  /**
+   * Object where all schemas are stored so that the table
+   * can reference to for dereferencing $ref schemas.
+   * Only necessary for ref rows.
+   */
```

Comment:
What should the object key be? A comment here would be helpful.

Reply:
Applied changes in #101